incoming-sms.php

```diff
@@ -18,11 +19,13 @@
     $config
 );
 
+$request = json_decode(file_get_contents('php://input'), true);
```

Comment:
Parses the FreeClimb Incoming Webhook Request through raw http post data

Reply:
@brgvgnsadd this as a code comment to the file above the `$request` line

Reply:
Should be in now!